training/tf/parse.py

```diff
@@ -19,47 +19,46 @@
 
 from tfprocess import TFProcess
 from chunkparser import ChunkParser
-import binascii
+import argparse
 import glob
 import gzip
-import itertools
-import math
 import multiprocessing as mp
-import numpy as np
-import queue
 import random
 import shufflebuffer as sb
-import struct
 import sys
 import tensorflow as tf
 import time
-import threading
-
-# 16 planes, 1 side to move, 1 x 362 probs, 1 winner = 19 lines
-DATA_ITEM_LINES = 16 + 1 + 1 + 1
 
 # Sane values are from 4096 to 64 or so. The maximum depends on the amount
 # of RAM in your GPU and the network size. You need to adjust the learning rate
 # if you change this.
 BATCH_SIZE = 512
 
-# Use a random sample of 1/16th of the input data read. This helps
-# improve the spread of games in the shuffle buffer.
-DOWN_SAMPLE = 16
+# Use a random sample input data read. This helps improve the spread of
+# games in the shuffle buffer.
+DOWN_SAMPLE = 1
```

Comment:
Is this an appropriate default when using v1 data?

Reply:
You're quite right, it isn't! :)

I've reverted it back to 16. If you end up using the v2_write_training.py, it would be reasonable to drop this to 1.